src/main/java/org/mitre/fhir/HapiReferenceServerProperties.java

```diff
@@ -232,7 +237,38 @@ public String getHibernateSearchLuceneVersion() {
         .getProperty(HIBERNATE_SEARCH_LUCENE_VERSION_KEY);
     return hibernateSearchLuceneVersion;
   }
+  
+  /**
+   * Returns the Hibernate Property.
```

Comment:
Is this property (and the two below) actually a `Hibernate` property?
```suggestion
   * Returns the public Client ID Property.
```